README.md

```diff
@@ -28,14 +29,21 @@ return [
 
 ## Using
 
-First all we need to set up the folder where the spec is be storing.
-This is the base folder relative for which we will structure the specification files.
+First of all, it's necessary to set up the folders where the spec is stored and where the generated doc must be placed.
+
+The api documentation we write is all inside `config_folder`.
+
+The generated html file is going to be inside `doc_folder`.
+
 ```yaml
 swagger:
-    config_folder: '%kernel.project_dir%/docs/api/'
+  apis:
+    frontend:
+      config_folder: '%kernel.project_dir%/docs/api/frontend/'
+      doc_folder: '%kernel.project_dir%/public/api/frontend/'
 ```
 
-Imagine you have a Swagger spec like this:
+Imagine you are to have a Swagger spec like this:
```

Comment:
previous variant is correct. this one sounds wrong

Reply:
The meaning is slightly different. What is meant here could be rephrased in following way:

```
Imagine that you need to have following Swagger specification:
```

If it's ok with you, I'll use this description, if not - previous variant will be kept